scripts/spawn-swarm.js

```diff
@@ -0,0 +1,12 @@
+const {swarmManager} = require('../src/swarmManager');
+require('../tests/test.configurations');
+
+(async () => {
+
+    const manager = await swarmManager();
+    const swarm = await manager.generateSwarm({numberOfDaemons: process.argv[2] || 3});
+    console.log(`Deployed ESR Contract to: ${manager.getEsrContractAddress()}`);
+    await manager.startAll();
+    console.log('Swarm started')
+
+})().catch(err => console.error(err));
```

Comment:
Let's use `console.trace` here instead so we can get the stack if needed.

Reply:
Separated out changes into separate PR. Resolved there. https://github.com/bluzelle/qa/pull/53